sr_robot_launch/launch/sr_ur10arm_box.launch

```diff
@@ -11,7 +11,7 @@
   <arg name="planning" default="ompl"/>
   <!-- Set to false if you want to start without guis -->
   <arg name="gui" default="true"/>
-  <arg name="verbose" default="true"/>
+  <arg name="verbose" default="true" />
```

Comment:
```suggestion
  <arg name="verbose" default="true"/>
```

Reply:
Fixed in https://github.com/shadow-robot/sr_interface/pull/419/commits/69bc13be8e64f769c2179bda3d2bfc388b4bc071